### PR TITLE
Sage/getting-started-on-celo-with-hardhat

### PR DESCRIPTION
**Pull request by bobeu**
Tutorial completed!

### CODE REVIEW

File: blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+## Basic Hardhat Configuration
+
+**What is Hardhat?**
+
+As defined on the official website, _Hardhat is a development environment for Ethereum software. It consists of different components for editing, compiling, debugging and deploying your smart contracts and dApps, all of which work together to create a complete development environment._
+
+Basically, Hardhat was built to target blockchain platforms that are compatible with Ethereum Virtual Machine - EVM, of which Celo happens to be one. Its flexibility of use and extendability makes it one of a kind. Hardhat potentially allows the installation of plugins, and it can even be configured to use other competitive tools, such as the Truffle framework. Let’s do a basic Hardhat installation and set up; to automate connection to Celo networks. As a reminder, you should have the aforementioned prerequisites set up before attempting this section.
+
+Open up a git bash terminal and the system terminal side by side. We don’t need two command lines, but I want to show you a common challenge most web3 developers often encounter and a simple hack that saves you the headache.
+![image](images/4.png)
```
Comment (viral-sangani) [inline blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md:47]: Add a new line for the image.

File: blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+```bash
+npx hardhat run scripts/deploy.js –network alfajores
+
+```
+
+The above command starts by invoking node package manager - **_npm_**, which goes into the `node_modules` (a folder containing all of the dependencies we need to run the program successfully), looks for hardhat, checks for **“run”** in its command list, invoke it on **“deploy.js”** file and instruct hardhat to deploy the outcome of the file to the selected network **“alfajores”** which is Celo testnet. 
+
+![image](images/7.png)
+
+`Note`: Hardhat will always perform a new compilation instruction if changes to previously compiled contract (s) exist. This is done using the cache file that will be generated soon as hardhat command is invoked.
```
Comment (viral-sangani) [inline blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md:147]:
```suggestion
> Note: Hardhat will always perform a new compilation instruction if changes to previously compiled contract (s) exist. This is done using the cache file that will be generated soon as hardhat command is invoked.
```

File: blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+
+Here, we are installing the Hardhat setup script as a dependency. When the installation is complete, we will create an instance of a hardhat boilerplate project. To do this, run:
+
+```bash
+npx hardhat
+
+```
+
+This command initializes a new instance of the Hardhat development environment. 
+`Note`: If you run `'npx hardhat'` using Bash CLI, you get a weird error like this.
```
Comment (viral-sangani) [inline blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md:77]:
```suggestion

> Note: If you run `'npx hardhat'` using Bash CLI, you get a weird error like this.
```

**Review by viral-sangani — CHANGES_REQUESTED** [review]

**Comment by viral-sangani** [comment]
@bobeu I have requested some minor changes. Can you please review this?

File: blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md (hunk 1 above)
Comment (bobeu) [reply blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md:48]: Resolved

Comment (bobeu) [reply blog/08-12-22-getting-started-on-celo-with-hardhat/gettng-started-on-celo-with-hardhat.md:48]: Fixed

**Review by viral-sangani — APPROVED** [review]

**Comment by viral-sangani** [comment]
LGTM.
@bobeu Congratulations on completing this article. 🎉

**Comment by bobeu** [comment]
> LGTM. @bobeu Congratulations on completing this article. 🎉

Thank you @viral-sangani

**Pull request merged**